### PR TITLE
PMM-6864 missing old metrics

### PR DESCRIPTION
**Pull request by Dasio**
[PMM-6864](https://jira.percona.com/browse/PMM-6864)
I tried to fill every metric needed for https://pmmdemo.percona.com/graph/d/mongodb-replicaset-summary/mongodb-replset-summary?orgId=1&refresh=1m

But this one is still missing: `mongodb_tcmalloc_cache_bytes`

- [ ] Links to other linked pull requests (optional).
---
FB: https://github.com/Percona-Lab/pmm-submodules/pull/1720
- [ ] Tests passed.
- [ ] Fix conflicts with target branch.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

When all checks have passed and code is ready for the review, bot should add `pmm-review-exporters` team as the reviewer. That would assign people from the review team automatically. Report any issues on our [Forums](https://forums.percona.com) and [Discord](https://discord.gg/mQEyGPkNbR).

### CODE REVIEW

File: exporter/v1_compatibility.go [hunk 1]
```diff
@@ -815,96 +819,162 @@ func myState(ctx context.Context, client *mongo.Client) (prometheus.Metric, erro
...
+	return []prometheus.Metric{headMetric, tailMetric}, nil
+
 }
 
-func replicationLag(m bson.M) prometheus.Metric {
-	var primaryTS, selfTS primitive.Timestamp
-	var hasPrimary bool
-	var name, statestr string
+// replSetStatus keeps the data returned by the GetReplSetStatus command.
+type replSetStatus struct {
```
Comment (percona-csalguero) [inline exporter/v1_compatibility.go:862]: We already have these structures in Toolkit: https://github.com/percona/percona-toolkit/blob/3.x/src/go/mongolib/proto/replstatus.go.
Can we use them instead?

**Review by percona-csalguero — CHANGES_REQUESTED** [review]

File: exporter/v1_compatibility.go (hunk 1 above)
Comment (Dasio) [reply exporter/v1_compatibility.go:862]: pingMS, lastHeartbeat and lastHeartbeatRecv is missing from there.

Comment (percona-csalguero) [reply exporter/v1_compatibility.go:862]: Update them as well.

Comment (Dasio) [reply exporter/v1_compatibility.go:862]: Did PR there https://github.com/percona/percona-toolkit/pull/490

Comment (percona-csalguero) [reply exporter/v1_compatibility.go:862]: Thanks. I've merged it. Could you update the dependency?

**Pull request merged**